packages/init/bootstrap-starter/near-runner/__tests__/main.ava.ts

```diff
@@ -0,0 +1,162 @@
+ *         }
+
+ *         pub fn get_status(&self, account_id: String) -> Option<String> {
+ *             return self.records.get(&account_id);
+ *         }
+ *     }
+ *
+ * As you can see, this contract only has two methods, a setter and a getter.
+ * The setter sets a free-form status message for the account that called the
+ * contract. The getter accepts an `account_id` param and returns the status for
+ * that account.
+ *
+ * The tests below create a local blockchain with this contract deployed to
+ * one account and two more accounts which store statuses in the contract.
+ *
+ *   [1]: https://github.com/near-examples/rust-status-message/tree/4e4767db257b748950bb3393352e2fff6c8e9b17
+ */
+
+/**
+ * Start off by importing Runner from near-runner-ava. This is usually the only
+ * import you'll need.
+ */
+import {Runner} from 'near-runner-ava';
+
+/**
+ * Create a new runner. In local sandbox mode, this will:
+ *
+ *   - Create a new local blockchain
+ *   - Create the root account for that blockchain (see `root` below)
+ *   - Execute any actions passed to the function
+ *   - Shut down the newly created blockchain, but *save the data*
+ */
+const runner = Runner.create(async ({root}) => {
+  // Create a subaccount of the root account, like `alice.sandbox`
+  // (the actual account name is not guaranteed; you can get it with `alice.accountId`)
+  const alice = await root.createAccount('alice');
+
+  // Create a subaccount of the root account, and also deploy a contract to it
+  const contract = await root.createAndDeploy(
+    // Subaccount name
+    'status-message',
+
+    // Relative path (from package.json location) to the compiled contract file
+    // which will be deployed to this account
+    'compiled-contracts/status_message.wasm',
+
+    // Provide `method` and `args` to call in the same block as the deploy
+    // {
+    //   method: 'init',
+    //   args: {owner_id: root},
+    // },
+  );
+
+  // You can call any other contract methods that you want to be executed at the
+  // beginning of all subsequent tests. In this example, Alice sets her status.
+  // Don't forget to `await` your calls!
+  await alice.call(contract, 'set_status', {message: 'hello'});
+
+  // Return the accounts that you want available in subsequent tests
+  // (`root` is always available)
+  return {alice, contract};
+});
+
+/**
+ * Now you can write some tests! In local sandbox mode, each `runner.test` will:
+ *
+ *   - start a new local blockchain
+ *   - copy the state from the blockchain created in `Runner.create`
+ *   - get access to the accounts created in `Runner.create` using the same variable names
+ *   - run concurrently with all other `runner.test` calls, keeping data isolated
+ *   - shut down at the end, forgetting all new data created
+ *
+ * It's also worth knowing that `runner.test` is syntax sugar added by
+ * near-runner-ava. With raw AVA + near-runner, here's how to write a test:
+ *
```

Comment:
```suggestion
 * 
 * ```ts
```

Reply:
I find the "indentation-only"-style code blocks more readable in plaintext format than the "fenced"-style codeblocks; we're already in a TS file here so I think it's implied.